openc3/lib/openc3/packets/packet_item.rb

```diff
@@ -489,35 +484,33 @@ def as_json(*a)
       config['read_conversion'] = self.read_conversion.as_json(*a) if self.read_conversion
       config['write_conversion'] = self.write_conversion.as_json(*a) if self.write_conversion
 
-      if self.limits
-        config['limits'] ||= {}
-        if self.limits.enabled
-          config['limits']['enabled'] = true
-        else
-          config['limits']['enabled'] = false
+      config['limits'] ||= {}
+      if self.limits.enabled
+        config['limits']['enabled'] = true
+      elsif self.limits.values or self.state_colors
+        # Only set to false if there are limits or state colors
+        # to avoid items without limits acting like they can be enabled
+        config['limits']['enabled'] = false
+      end
+      if self.limits.values
+        # Only set these if there are limits.values because persistence_setting has a default
+        # and we don't want keys on the 'limits' hash if there aren't any limits
+        config['limits']['persistence_setting'] = self.limits.persistence_setting if self.limits.persistence_setting
+        config['limits']['response'] = self.limits.response.to_s if self.limits.response
+        self.limits.values.each do |limits_set, limits_values|
+          limits = {}
+          limits['red_low'] =  limits_values[0]
+          limits['yellow_low'] = limits_values[1]
+          limits['yellow_high'] = limits_values[2]
+          limits['red_high'] = limits_values[3]
+          limits['green_low'] = limits_values[4] if limits_values[4]
+          limits['green_high'] = limits_values[5] if limits_values[5]
+          config['limits'][limits_set] = limits
         end
-        if self.limits.values
-          config['limits'] ||= {}
-          config['limits']['persistence_setting'] = self.limits.persistence_setting
-          config['limits']['response'] = self.limits.response.to_s if self.limits.response
-          self.limits.values.each do |limits_set, limits_values|
-            limits = {}
-            limits['red_low'] =  limits_values[0]
-            limits['yellow_low'] = limits_values[1]
-            limits['yellow_high'] = limits_values[2]
-            limits['red_high'] = limits_values[3]
-            limits['green_low'] = limits_values[4] if limits_values[4]
-            limits['green_high'] = limits_values[5] if limits_values[5]
-            config['limits'][limits_set] = limits
-          end
-        end
-        config['limits_response'] = self.limits.response.as_json(*a) if self.limits.response
```

Comment:
This key shouldn't have been used ... it's already at `config['limits']['response']`